config/config_performance.yaml

```diff
@@ -9,15 +9,14 @@ kraken:
     litmus_uninstall: False                                # If you want to uninstall litmus if failure
     litmus_uninstall_before_run: True                      # If you want to uninstall litmus before a new run starts
     chaos_scenarios:                                       # List of policies/chaos scenarios to load
-        -   pod_scenarios:                                 # List of chaos pod scenarios to load
-            - -    scenarios/openshift/etcd.yml
-            - -    scenarios/openshift/regex_openshift_pod_kill.yml
-              -    scenarios/openshift/post_action_regex.py
+        -   plugin_scenarios:                                 # List of chaos pod scenarios to load
+            - scenarios/openshift/etcd.yml
+            - scenarios/openshift/regex_openshift_pod_kill.yml
```

Comment:
With taking out the double array here, we are taking out the functionality to be able to pass a personalized script (not in the scenario yaml file itself) to verify the pods have recovered properly. 
@chaitanyaenr what are your thoughts on taking this out? I feel that a lot of users have brought this up as a nice to have functionality

Reply:
the double array allows a user to pass a secondary script to do the pre/post action to validate the number of pods. 
so for the openshift regex scenario, there is no way to know the number of pods that match the namespace and label on any given openshift cluster (will depend on their certain cluster configuration). So we added a way to pass a personalized script to get the number of pods that exist before the scenario runs and then validate that number after the scenario runs to have resiliency check.

Reply:
Agree that the generic checks can be part of a plugin but there might be some use cases where user wants to run few quick checks that are very specific to their service or test case which they might not be wanting to write a plugin for, so it's better to still support a way for user to plugin in their own checks or if the eventual engine workflow doesn't support it then we can recommend running the checks outside Kraken - something we can discuss I guess. Maybe we can add a note about plugin scenarios not supporting it for now and discuss on the best way moving forward. Thoughts?